packages/client/src/comp/states/Base.test.tsx

```diff
@@ -214,3 +214,49 @@ test('Signup Flow allows for separate mailing address', async () => {
   await wait(() => expect(register).toHaveBeenCalledTimes(1))
   await compareResults(register, true)
 })
+
+test("Sends fax to twillion successfully", async () => {
```

Comment:
To test this, you need to select a fax-only state like Maine: (see `stateMethods`).

Nit: "Twillion" should be Twillio.